### PR TITLE
Update the documentation with comments for playing YouTube Live Streams

### PR DESCRIPTION
**Pull request by skinkie**

### CODE REVIEW

File: docs/players.rst [hunk 1]
```diff
@@ -95,7 +95,15 @@ The stable version of VLC cannot play Youtube Live streams. This a known bug
...
+Some versions of Mplayer cannot play Youtube Live streams. And errors like:
+Cannot seek backward in linear streams!
+Seek failed
+Switching to a recent fork such as mpv resolves the issue.
+
 When using OMXPlayer the stream stops unexpectedly
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 When reading from a fifo pipe OMXPlayer will quit when there is no data, to fix
 this you can supply the timeout option to OMXPlayer using :option:`--player "omxplayer --timeout 20" <--player>`.
+For live streams it might be beneficial to also add :option:`--live`.
```
Comment (bastimeyer) [inline docs/players.rst]: https://travis-ci.org/streamlink/streamlink/jobs/452411724#L1174

The `:option:` directive is only meant for Streamlink parameters:
https://github.com/streamlink/streamlink/blob/master/docs/ext_argparse.py#L82

Use this instead for regular parameters:
```
``--live``
```

File: docs/players.rst [hunk 2]
```diff
@@ -95,7 +95,15 @@ The stable version of VLC cannot play Youtube Live streams. This a known bug
 that is fixed in VLC 3, which is still in development. You can also try
 using a different player.
 
+Youtube Live does not work with Mplayer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some versions of Mplayer cannot play Youtube Live streams. And errors like:
+Cannot seek backward in linear streams!
```
Comment (bastimeyer) [inline docs/players.rst]:
```suggestion

.. code-block:: console

    Cannot seek backward in linear streams!
    Seek failed

```

If you set the error messages into a code-block for console output (mind the white space), it'll make it more distinguishable from the regular text.

**Comment by bastimeyer** [comment]
https://travis-ci.org/streamlink/streamlink/jobs/452466332#L1169
The empty lines are required. Take a look at my suggestion again:
https://github.com/streamlink/streamlink/pull/2156#discussion_r231930400

```rst
Youtube Live does not work with Mplayer
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Some versions of Mplayer cannot play Youtube Live streams and return errors like:

.. code-block:: console

    Cannot seek backward in linear streams!
    Seek failed

Switching to a recent fork such as mpv resolves the issue.
```

**Comment by gravyboat** [comment]
Looks good now, thanks.

**Pull request merged**